### PR TITLE
 sql-plan-replayer: add doc for historical stats integration into the plan replayer

### PR DESCRIPTION
**Pull request by time-and-fate**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):
https://github.com/pingcap/tidb/issues/45038
https://github.com/pingcap/tidb/pull/44592

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-plan-replayer.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ TiDB 根据 `sql-statement` 整理出以下集群现场信息：
 - `EXPLAIN [ANALYZE] sql-statement` 的结果
 - 优化器进行查询优化的一些内部步骤的记录
 
+当历史统计信息被[启用](/system-variables.md#tidb_enable_historical_stats)时，可以在 `PLAN REPLAYER` 语句中指定时间来获取对应时间的统计信息。该语法支持直接指定日期时间或指定时间戳。此时，TiDB 会寻找指定时间之前的历史统计信息，并导出其中最新的一份。
```
Comment (ran-huang) [inline sql-plan-replayer.md:36]:
```suggestion
当[启用历史统计信息](/system-variables.md#tidb_enable_historical_stats)时，可以在 `PLAN REPLAYER` 语句中指定时间来获取对应时间的统计信息。该语法支持直接指定日期时间或指定时间戳。此时，TiDB 会寻找指定时间之前的历史统计信息，并导出其中最新的一份。
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

**Review by qw4990 — APPROVED** [review]

File: sql-plan-replayer.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ TiDB 根据 `sql-statement` 整理出以下集群现场信息：
 - `EXPLAIN [ANALYZE] sql-statement` 的结果
 - 优化器进行查询优化的一些内部步骤的记录
 
+当[启用历史统计信息](/system-variables.md#tidb_enable_historical_stats)时，可以在 `PLAN REPLAYER` 语句中指定时间来获取对应时间的统计信息。该语法支持直接指定日期时间或指定时间戳。此时，TiDB 会寻找指定时间之前的历史统计信息，并导出其中最新的一份。
```
Comment (qiancai) [inline sql-plan-replayer.md:36]:
```suggestion
当[启用历史统计信息](/system-variables.md#tidb_enable_historical_stats)时，可以在 `PLAN REPLAYER` 语句中指定时间来获取对应时间的统计信息。该语法支持直接指定日期时间或指定时间戳。此时，TiDB 会查找指定时间之前的历史统计信息，并导出其中最新的一份。
```

File: sql-plan-replayer.md [hunk 3]
```diff
@@ -33,6 +33,10 @@ TiDB 根据 `sql-statement` 整理出以下集群现场信息：
...
+
+如果没有符合指定时间的历史统计信息，TiDB 会直接导出最新统计信息，也就是和未指定时间时的行为一致，并且在导出的 `ZIP` 文件中的 `errors.txt` 中输出错误信息。
```
Comment (qiancai) [inline sql-plan-replayer.md:38]:
```suggestion
如果没有指定时间之前的历史统计信息，TiDB 会直接导出最新统计信息，也就是和未指定时间时的行为一致，并且在导出的 `ZIP` 文件中的 `errors.txt` 中输出错误信息。
```

Comment (qiancai) [inline sql-plan-replayer.md:38]:
```suggestion
如果没有指定时间之前的历史统计信息，TiDB 会直接导出最新统计信息，也就是和未指定时间时的行为一致，并且在导出的 `ZIP` 文件中的 `errors.txt` 中输出错误信息。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**